packages/typescript/src/options/plugin.ts

```diff
@@ -14,7 +14,7 @@ import getTsLibPath from '../tslib';
  * - `typescript`: Instance of Typescript library (possibly custom).
  * - `tslib`: ESM code from the tslib helper library (possibly custom).
  */
-export default function getPluginOptions(options: RollupTypescriptOptions) {
+export const getPluginOptions = (options: RollupTypescriptOptions) => {
```

Comment:
Did we change linter rules to ban default exports?

Reply:
Just changed the eslint config so that it didn't demand default exports. I hate em.

Reply:
Cool, lots of the default exports here were only to appease the linter :)